registry.json

```diff
@@ -3898,5 +3898,19 @@
     "telegram": "lukas_psya",
     "operatorName": "",
     "github": "rafek47"
+  },
+  {
+    "ticker": "5HARE",
+    "poolId": "73a527af03d38b3659f20b263000a1f72d3f3b867df5a2bb0e5516f1",
+    "social": {
+      "website": "https://5hare-pool.5pecia1.dev",
+      "twitter": "https://twitter.com/5HARE_Pool",
```

Comment:
There is no activity on this account... We look for some amount of active social media presence. Also please can you make this just the twitter handle, e.g. `5HARE_Pool`